### DISCUSSION TITLE
On-screen controls with hud

### DISCUSSION THREAD

**Discussion by Willian514** [redphx/better-xcloud#205] [General]
It would be cool to add this extension to the mobile version, as the controls that are natively activated on the Xbox, it is not possible to completely change the hud, test it using the Kiwi browser 
[Extensão Touch ### xCloud.zip](https://github.com/redphx/better-xcloud/files/13835836/Extensao.Touch.xCloud.zip)

https://github.com/redphx/better-xcloud/assets/155685896/beaa2880-399a-4b0a-89c4-87a90115f920

Currently the extension no longer works for me, but the scheme is this

**Comment by redphx** [comment]
The current touch controller of xCloud doesn't support moving buttons separately

**Reply by Willian514** [reply]
I know that, but with this extension it is possible to move the control hud

**Reply by Willian514** [reply]
And on Android TV, the controls don't appear on the screen for me to configure the key mapper, I mean xcloud native

**Reply by KingNothing81** [reply]
> And on Android TV, the controls don't appear on the screen for me to configure the key mapper, I mean xcloud native

MKB support doesn’t work on Android and iOS. So with version 3.0.1, the feature won’t show up anymore on those unsupported platforms.

https://github.com/redphx/better-xcloud/issues/206
